pkgs/os-specific/linux/brillo/default.nix

```
@@ -0,0 +1,28 @@
+{stdenv, fetchFromGitLab , which, go-md2man, coreutils}:
+
+stdenv.mkDerivation rec {
+  version = "1.4.3";
+  name = "brillo-${version}";
```

The following is enough nowadays:

```suggestion
  pname = "brillo";
```